cli/src/config.ts

```diff
@@ -193,7 +194,17 @@ export class Config implements CliConfig {
   }
 
   private initWindowsConfig() {
-    this.windows.androidStudioPath = this.app.windowsAndroidStudioPath && this.app.windowsAndroidStudioPath;
+    if (this.app.windowsAndroidStudioPath) {
+      if (!existsSync(this.app.windowsAndroidStudioPath) && this.cli.os === OS.Windows) {
```

Comment:
can you put the `this.cli.os === OS.Windows` first? in that case it won't try to see if the file exists in non windows

Reply:
An alternative is to switch the comparison inside the if
If the first operand is false, JavaScript will short-circuit and not evaluate the second condition.
```
if (this.cli.os === OS.Windows && !existsSync(this.app.windowsAndroidStudioPath)) {
```

Reply:
Yeah, that's what I meant when I said first, in the same if, but it's also ok to just do nothing if it's not windows as Genarito suggested.